upi/openstack/update_machineset.yaml

```diff
@@ -0,0 +1,57 @@
+# Required Python packages:
```

Comment:
we should consider invoking this automatically (say after the control plane is UP)